src/pykeen/hpo/__init__.py

```diff
@@ -23,9 +23,9 @@
 
 >>> from pykeen.hpo import hpo_pipeline
 >>> hpo_pipeline_result = hpo_pipeline(
-...    timeout=60,
-...    dataset='Nations',
-...    model='TransE',
+...     timeout=60,
```

Comment:
white-space change? It seems to be related to continued lines for code example blocks in docstring. there are quite a few, but I only added a comment here.

Reply:
Before it was 3 space indentation, which is not the norm